pennylane/pulse/hardware_hamiltonian.py

```diff
@@ -206,74 +207,55 @@ def circuit(params):
         Array(0.94301294, dtype=float64)
         >>> jax.grad(circuit)(params)
         [Array(0.59484969, dtype=float64)]
```

Comment:
These examples will likely produce different results now that detuning isn't included in `drive` the same way it was before.